entgql/template/where_input.tmpl

```diff
@@ -17,6 +17,7 @@ in the LICENSE file in the root directory of this source tree.
 
 {{- range $n := filterNodes $.Nodes }}
     {{ $comparableFields := list }}
+    {{ $comparableFields = append $comparableFields $n.ID }}
```

Comment:
I'm not sure if `id` should be filterable by `filterFields`

Reply:
We should make it possible to use the `Skip` annotation on ID field, but this can be done in future PRs.